types/schemas/postSubmissionApplication/README.md

```diff
@@ -0,0 +1,296 @@
+# Post Submission Application Schema
+
+The Post Submission Application schema is designed to represent an application, post-submission, within an application.
+
+Future iterations will create redacted versions of this, and the `PrototypeApplication` schema, which will act as a 'published' schema for applications such as the [DPR](https://planningregister.org/).
+
+## The Application Process
+
+```mermaid
```

Comment:
:mermaid: Appreciate a good README, thank you!! Nice motivation to add similar in other `types/schemas/` subdirectories too